linode/manager
# feat: [M3-6971] Add dynamic pricing model to Volume creation flows

## Description 📝
This PR adds dynamic (DC based) pricing to the Volume creation flows. 

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-08-21 at 00 10 21](https://github.com/linode/manager/assets/130582365/d74c442c-121c-49bf-900d-a07f68ff7097) | ![Screenshot 2023-08-21 at 00 09 08](https://github.com/linode/manager/assets/130582365/ce3ba37f-be01-43d0-a6c2-4a005982a400) |
| ![Screenshot 2023-08-21 at 00 10 39](https://github.com/linode/manager/assets/130582365/177da3d4-91f6-44f9-84b2-448ac4cdb286) | ![Screenshot 2023-08-21 at 00 09 40](https://github.com/linode/manager/assets/130582365/53cc3be5-6e90-4c2c-a829-04410f642571) |

## How to test 🧪
1. Check out code locally and enable the `DC-Specific Pricing` feature flag in the dev settings
2. Test the following flows with both the flag on and off. 
- The off flag should match production (except for some styling fixes -- see self review) and the on flag the dynamic pricing implementation
- With the flag on, flow should be tested with Sao Paulo, Jakarta and and any region without a price increase

**Flows**:
- /volumes/create
- /volumes => action menu: resize & clone
- /linodes/[id]/storage (w/ linodes in both regular and increased priced regions): create volume + action menu: resize & clone with attached volumes
